[1;92mINFO [2024-04-20 17:25:03 - connection.py:47] - Connecting...[0m
[1;92mINFO [2024-07-06 12:12:16 - auth.py:82] - Start creating a new auth key on DC2[0m
[1;92mINFO [2024-07-06 12:12:16 - tcp.py:67] - Using proxy 127.0.0.1[0m
[1;92mINFO [2024-07-06 12:12:16 - connection.py:47] - Connecting...[0m
[1;92mINFO [2024-07-06 12:12:16 - connection.py:54] - Connected! Production DC2 - IPv4[0m
[1;92mINFO [2024-07-06 12:12:18 - auth.py:267] - Done auth key exchange: DhGenOk[0m
[1;92mINFO [2024-07-06 12:12:18 - connection.py:66] - Disconnected[0m
[1;92mINFO [2024-07-06 12:12:18 - tcp.py:67] - Using proxy 127.0.0.1[0m
[1;92mINFO [2024-07-06 12:12:18 - connection.py:47] - Connecting...[0m
[1;92mINFO [2024-07-06 12:12:18 - connection.py:54] - Connected! Production DC2 - IPv4[0m
[1;92mINFO [2024-07-06 12:12:18 - session.py:294] - NetworkTask started[0m
[1;92mINFO [2024-07-06 12:12:19 - session.py:139] - Session initialized: Layer 158[0m
[1;92mINFO [2024-07-06 12:12:19 - session.py:140] - Device: CPython 3.10.12 - Pyrogram 2.0.106[0m
[1;92mINFO [2024-07-06 12:12:19 - session.py:141] - System: Linux 5.15.0-101-generic (en)[0m
[1;92mINFO [2024-07-06 12:12:19 - session.py:155] - Session started[0m
[1;92mINFO [2024-07-06 12:12:19 - session.py:272] - PingTask started[0m
[1;92mINFO [2024-07-06 12:12:19 - session.py:291] - PingTask stopped[0m
[1;92mINFO [2024-07-06 12:12:19 - connection.py:66] - Disconnected[0m
[1;92mINFO [2024-07-06 12:12:19 - session.py:315] - NetworkTask stopped[0m
[1;92mINFO [2024-07-06 12:12:19 - session.py:180] - Session stopped[0m
[1;92mINFO [2024-07-06 12:12:19 - auth.py:82] - Start creating a new auth key on DC4[0m
[1;92mINFO [2024-07-06 12:12:19 - tcp.py:67] - Using proxy 127.0.0.1[0m
[1;92mINFO [2024-07-06 12:12:19 - connection.py:47] - Connecting...[0m
[1;92mINFO [2024-07-06 12:12:20 - connection.py:54] - Connected! Production DC4 - IPv4[0m
[1;92mINFO [2024-07-06 12:12:22 - auth.py:267] - Done auth key exchange: DhGenOk[0m
[1;92mINFO [2024-07-06 12:12:22 - connection.py:66] - Disconnected[0m
[1;92mINFO [2024-07-06 12:12:22 - tcp.py:67] - Using proxy 127.0.0.1[0m
[1;92mINFO [2024-07-06 12:12:22 - connection.py:47] - Connecting...[0m
[1;92mINFO [2024-07-06 12:12:22 - connection.py:54] - Connected! Production DC4 - IPv4[0m
[1;92mINFO [2024-07-06 12:12:22 - session.py:294] - NetworkTask started[0m
[1;92mINFO [2024-07-06 12:12:23 - session.py:139] - Session initialized: Layer 158[0m
[1;92mINFO [2024-07-06 12:12:23 - session.py:140] - Device: CPython 3.10.12 - Pyrogram 2.0.106[0m
[1;92mINFO [2024-07-06 12:12:23 - session.py:141] - System: Linux 5.15.0-101-generic (en)[0m
[1;92mINFO [2024-07-06 12:12:23 - session.py:155] - Session started[0m
[1;92mINFO [2024-07-06 12:12:23 - session.py:272] - PingTask started[0m
[1;92mINFO [2024-07-06 12:12:23 - client.py:698] - [vid-compressbot] [LOAD] MessageHandler("say_hello") in group 0 from "plugins.start"[0m
[1;92mINFO [2024-07-06 12:12:23 - client.py:775] - [vid-compressbot] Successfully loaded 1 plugin from "plugins"[0m
[1;92mINFO [2024-07-06 12:12:23 - dispatcher.py:154] - Started 8 HandlerTasks[0m
[1;92mINFO [2024-07-06 12:12:27 - idle.py:75] - Stop signal received (SIGINT). Exiting...[0m
[1;92mINFO [2024-07-06 12:13:00 - tcp.py:67] - Using proxy 127.0.0.1[0m
[1;92mINFO [2024-07-06 12:13:00 - connection.py:47] - Connecting...[0m
[1;92mINFO [2024-07-06 12:13:01 - connection.py:54] - Connected! Production DC4 - IPv4[0m
[1;92mINFO [2024-07-06 12:13:01 - session.py:294] - NetworkTask started[0m
[1;92mINFO [2024-07-06 12:13:01 - session.py:139] - Session initialized: Layer 158[0m
[1;92mINFO [2024-07-06 12:13:01 - session.py:140] - Device: CPython 3.10.12 - Pyrogram 2.0.106[0m
[1;92mINFO [2024-07-06 12:13:01 - session.py:141] - System: Linux 5.15.0-101-generic (en)[0m
[1;92mINFO [2024-07-06 12:13:01 - session.py:155] - Session started[0m
[1;92mINFO [2024-07-06 12:13:01 - session.py:272] - PingTask started[0m
[1;92mINFO [2024-07-06 12:13:01 - client.py:698] - [vid-compressbot] [LOAD] MessageHandler("say_hello") in group 0 from "plugins.start"[0m
[1;92mINFO [2024-07-06 12:13:01 - client.py:775] - [vid-compressbot] Successfully loaded 1 plugin from "plugins"[0m
[1;92mINFO [2024-07-06 12:13:01 - dispatcher.py:154] - Started 8 HandlerTasks[0m
[1;92mINFO [2024-07-06 12:13:04 - idle.py:75] - Stop signal received (SIGINT). Exiting...[0m
[1;92mINFO [2024-07-08 09:13:06 - auth.py:82] - Start creating a new auth key on DC2[0m
[1;92mINFO [2024-07-08 09:13:06 - tcp.py:67] - Using proxy 127.0.0.1[0m
[1;92mINFO [2024-07-08 09:13:06 - connection.py:47] - Connecting...[0m
[1;92mINFO [2024-07-08 09:13:06 - connection.py:54] - Connected! Production DC2 - IPv4[0m
[1;92mINFO [2024-07-08 09:13:08 - auth.py:267] - Done auth key exchange: DhGenOk[0m
[1;92mINFO [2024-07-08 09:13:08 - connection.py:66] - Disconnected[0m
[1;92mINFO [2024-07-08 09:13:08 - tcp.py:67] - Using proxy 127.0.0.1[0m
[1;92mINFO [2024-07-08 09:13:08 - connection.py:47] - Connecting...[0m
[1;92mINFO [2024-07-08 09:13:08 - connection.py:54] - Connected! Production DC2 - IPv4[0m
[1;92mINFO [2024-07-08 09:13:08 - session.py:294] - NetworkTask started[0m
[1;92mINFO [2024-07-08 09:13:08 - session.py:139] - Session initialized: Layer 158[0m
[1;92mINFO [2024-07-08 09:13:08 - session.py:140] - Device: CPython 3.10.12 - Pyrogram 2.0.106[0m
[1;92mINFO [2024-07-08 09:13:08 - session.py:141] - System: Linux 5.15.0-101-generic (en)[0m
[1;92mINFO [2024-07-08 09:13:08 - session.py:155] - Session started[0m
[1;92mINFO [2024-07-08 09:13:08 - session.py:272] - PingTask started[0m
[1;92mINFO [2024-07-08 09:13:09 - session.py:291] - PingTask stopped[0m
[1;92mINFO [2024-07-08 09:13:09 - connection.py:66] - Disconnected[0m
[1;92mINFO [2024-07-08 09:13:09 - session.py:315] - NetworkTask stopped[0m
[1;92mINFO [2024-07-08 09:13:09 - session.py:180] - Session stopped[0m
[1;92mINFO [2024-07-08 09:13:09 - auth.py:82] - Start creating a new auth key on DC4[0m
[1;92mINFO [2024-07-08 09:13:09 - tcp.py:67] - Using proxy 127.0.0.1[0m
[1;92mINFO [2024-07-08 09:13:09 - connection.py:47] - Connecting...[0m
[1;92mINFO [2024-07-08 09:13:09 - connection.py:54] - Connected! Production DC4 - IPv4[0m
[1;92mINFO [2024-07-08 09:13:10 - auth.py:267] - Done auth key exchange: DhGenOk[0m
[1;92mINFO [2024-07-08 09:13:10 - connection.py:66] - Disconnected[0m
[1;92mINFO [2024-07-08 09:13:10 - tcp.py:67] - Using proxy 127.0.0.1[0m
[1;92mINFO [2024-07-08 09:13:10 - connection.py:47] - Connecting...[0m
[1;92mINFO [2024-07-08 09:13:11 - connection.py:54] - Connected! Production DC4 - IPv4[0m
[1;92mINFO [2024-07-08 09:13:11 - session.py:294] - NetworkTask started[0m
[1;92mINFO [2024-07-08 09:13:12 - session.py:139] - Session initialized: Layer 158[0m
[1;92mINFO [2024-07-08 09:13:12 - session.py:140] - Device: CPython 3.10.12 - Pyrogram 2.0.106[0m
[1;92mINFO [2024-07-08 09:13:12 - session.py:141] - System: Linux 5.15.0-101-generic (en)[0m
[1;92mINFO [2024-07-08 09:13:12 - session.py:155] - Session started[0m
[1;92mINFO [2024-07-08 09:13:12 - session.py:272] - PingTask started[0m
[1;92mINFO [2024-07-08 09:13:13 - client.py:698] - [vid-compressbot] [LOAD] MessageHandler("say_hello") in group 0 from "plugins.start"[0m
[1;92mINFO [2024-07-08 09:13:13 - client.py:775] - [vid-compressbot] Successfully loaded 1 plugin from "plugins"[0m
[1;92mINFO [2024-07-08 09:13:13 - dispatcher.py:154] - Started 8 HandlerTasks[0m
[1;92mINFO [2024-07-08 09:13:37 - idle.py:75] - Stop signal received (SIGINT). Exiting...[0m
[1;92mINFO [2024-07-08 09:13:57 - tcp.py:67] - Using proxy 127.0.0.1[0m
[1;92mINFO [2024-07-08 09:13:57 - connection.py:47] - Connecting...[0m
[1;92mINFO [2024-07-08 09:13:57 - connection.py:54] - Connected! Production DC4 - IPv4[0m
[1;92mINFO [2024-07-08 09:13:57 - session.py:294] - NetworkTask started[0m
[1;92mINFO [2024-07-08 09:13:57 - session.py:139] - Session initialized: Layer 158[0m
[1;92mINFO [2024-07-08 09:13:57 - session.py:140] - Device: CPython 3.10.12 - Pyrogram 2.0.106[0m
[1;92mINFO [2024-07-08 09:13:57 - session.py:141] - System: Linux 5.15.0-101-generic (en)[0m
[1;92mINFO [2024-07-08 09:13:57 - session.py:155] - Session started[0m
[1;92mINFO [2024-07-08 09:13:57 - session.py:272] - PingTask started[0m
[1;92mINFO [2024-07-08 09:13:58 - client.py:698] - [vid-compressbot] [LOAD] MessageHandler("say_hello") in group 0 from "plugins.start"[0m
[1;92mINFO [2024-07-08 09:13:58 - client.py:775] - [vid-compressbot] Successfully loaded 1 plugin from "plugins"[0m
[1;92mINFO [2024-07-08 09:13:58 - dispatcher.py:154] - Started 8 HandlerTasks[0m
[1;92mINFO [2024-07-08 09:14:12 - idle.py:75] - Stop signal received (SIGINT). Exiting...[0m
[1;92mINFO [2024-07-08 09:14:21 - tcp.py:67] - Using proxy 127.0.0.1[0m
[1;92mINFO [2024-07-08 09:14:21 - connection.py:47] - Connecting...[0m
[1;92mINFO [2024-07-08 09:14:22 - connection.py:54] - Connected! Production DC4 - IPv4[0m
[1;92mINFO [2024-07-08 09:14:22 - session.py:294] - NetworkTask started[0m
[1;92mINFO [2024-07-08 09:14:22 - session.py:139] - Session initialized: Layer 158[0m
[1;92mINFO [2024-07-08 09:14:22 - session.py:140] - Device: CPython 3.10.12 - Pyrogram 2.0.106[0m
[1;92mINFO [2024-07-08 09:14:22 - session.py:141] - System: Linux 5.15.0-101-generic (en)[0m
[1;92mINFO [2024-07-08 09:14:22 - session.py:155] - Session started[0m
[1;92mINFO [2024-07-08 09:14:22 - session.py:272] - PingTask started[0m
[1;92mINFO [2024-07-08 09:14:23 - client.py:698] - [vid-compressbot] [LOAD] MessageHandler("say_hello") in group 0 from "plugins.start"[0m
[1;92mINFO [2024-07-08 09:14:23 - client.py:775] - [vid-compressbot] Successfully loaded 1 plugin from "plugins"[0m
[1;92mINFO [2024-07-08 09:14:23 - dispatcher.py:154] - Started 8 HandlerTasks[0m
[1;92mINFO [2024-07-08 09:14:36 - idle.py:75] - Stop signal received (SIGINT). Exiting...[0m
[1;92mINFO [2024-07-08 11:31:15 - auth.py:82] - Start creating a new auth key on DC2[0m
[1;92mINFO [2024-07-08 11:31:15 - tcp.py:67] - Using proxy 127.0.0.1[0m
[1;92mINFO [2024-07-08 11:31:15 - connection.py:47] - Connecting...[0m
[1;92mINFO [2024-07-08 11:31:15 - connection.py:54] - Connected! Production DC2 - IPv4[0m
[1;92mINFO [2024-07-08 11:31:17 - auth.py:267] - Done auth key exchange: DhGenOk[0m
[1;92mINFO [2024-07-08 11:31:17 - connection.py:66] - Disconnected[0m
[1;92mINFO [2024-07-08 11:31:17 - tcp.py:67] - Using proxy 127.0.0.1[0m
[1;92mINFO [2024-07-08 11:31:17 - connection.py:47] - Connecting...[0m
[1;92mINFO [2024-07-08 11:31:17 - connection.py:54] - Connected! Production DC2 - IPv4[0m
[1;92mINFO [2024-07-08 11:31:17 - session.py:294] - NetworkTask started[0m
[1;92mINFO [2024-07-08 11:31:18 - session.py:139] - Session initialized: Layer 158[0m
[1;92mINFO [2024-07-08 11:31:18 - session.py:140] - Device: CPython 3.10.12 - Pyrogram 2.0.106[0m
[1;92mINFO [2024-07-08 11:31:18 - session.py:141] - System: Linux 5.15.0-101-generic (en)[0m
[1;92mINFO [2024-07-08 11:31:18 - session.py:155] - Session started[0m
[1;92mINFO [2024-07-08 11:31:18 - session.py:272] - PingTask started[0m
[1;92mINFO [2024-07-08 11:31:18 - session.py:291] - PingTask stopped[0m
[1;92mINFO [2024-07-08 11:31:18 - connection.py:66] - Disconnected[0m
[1;92mINFO [2024-07-08 11:31:18 - session.py:315] - NetworkTask stopped[0m
[1;92mINFO [2024-07-08 11:31:18 - session.py:180] - Session stopped[0m
[1;92mINFO [2024-07-08 11:31:18 - auth.py:82] - Start creating a new auth key on DC4[0m
[1;92mINFO [2024-07-08 11:31:18 - tcp.py:67] - Using proxy 127.0.0.1[0m
[1;92mINFO [2024-07-08 11:31:18 - connection.py:47] - Connecting...[0m
[1;92mINFO [2024-07-08 11:31:18 - connection.py:54] - Connected! Production DC4 - IPv4[0m
[1;92mINFO [2024-07-08 11:31:20 - auth.py:267] - Done auth key exchange: DhGenOk[0m
[1;92mINFO [2024-07-08 11:31:20 - connection.py:66] - Disconnected[0m
[1;92mINFO [2024-07-08 11:31:20 - tcp.py:67] - Using proxy 127.0.0.1[0m
[1;92mINFO [2024-07-08 11:31:20 - connection.py:47] - Connecting...[0m
[1;92mINFO [2024-07-08 11:31:20 - connection.py:54] - Connected! Production DC4 - IPv4[0m
[1;92mINFO [2024-07-08 11:31:20 - session.py:294] - NetworkTask started[0m
[1;92mINFO [2024-07-08 11:31:21 - session.py:139] - Session initialized: Layer 158[0m
[1;92mINFO [2024-07-08 11:31:21 - session.py:140] - Device: CPython 3.10.12 - Pyrogram 2.0.106[0m
[1;92mINFO [2024-07-08 11:31:21 - session.py:141] - System: Linux 5.15.0-101-generic (en)[0m
[1;92mINFO [2024-07-08 11:31:21 - session.py:155] - Session started[0m
[1;92mINFO [2024-07-08 11:31:21 - session.py:272] - PingTask started[0m
[1;92mINFO [2024-07-08 11:31:21 - client.py:698] - [vid-compressbot] [LOAD] MessageHandler("say_hello") in group 0 from "plugins.start"[0m
[1;92mINFO [2024-07-08 11:31:21 - client.py:775] - [vid-compressbot] Successfully loaded 1 plugin from "plugins"[0m
[1;92mINFO [2024-07-08 11:31:21 - dispatcher.py:154] - Started 8 HandlerTasks[0m
[1;91mERROR [2024-07-08 11:31:31 - dispatcher.py:253] - Error -2 connecting to redis:6379. Name or service not known.[0m
Traceback (most recent call last):
  File "/home/freeguy/.local/lib/python3.10/site-packages/redis/connection.py", line 264, in connect
    sock = self.retry.call_with_retry(
  File "/home/freeguy/.local/lib/python3.10/site-packages/redis/retry.py", line 46, in call_with_retry
    return do()
  File "/home/freeguy/.local/lib/python3.10/site-packages/redis/connection.py", line 265, in <lambda>
    lambda: self._connect(), lambda error: self.disconnect(error)
  File "/home/freeguy/.local/lib/python3.10/site-packages/redis/connection.py", line 595, in _connect
    for res in socket.getaddrinfo(
  File "/usr/lib/python3.10/socket.py", line 955, in getaddrinfo
    for res in _socket.getaddrinfo(host, port, family, type, proto, flags):
socket.gaierror: [Errno -2] Name or service not known

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/freeguy/.local/lib/python3.10/site-packages/pyrogram/dispatcher.py", line 240, in handler_worker
    await handler.callback(self.client, *args)
  File "/home/freeguy/Desktop/project/vid-compress/app/plugins/start.py", line 9, in say_hello
    data = con.setting
  File "/home/freeguy/Desktop/project/vid-compress/app/utils/connection.py", line 32, in setting
    last_request_time = self.redis_client.get('last_request_time')
  File "/home/freeguy/.local/lib/python3.10/site-packages/redis/commands/core.py", line 1829, in get
    return self.execute_command("GET", name)
  File "/home/freeguy/.local/lib/python3.10/site-packages/redis/client.py", line 533, in execute_command
    conn = self.connection or pool.get_connection(command_name, **options)
  File "/home/freeguy/.local/lib/python3.10/site-packages/redis/connection.py", line 1086, in get_connection
    connection.connect()
  File "/home/freeguy/.local/lib/python3.10/site-packages/redis/connection.py", line 270, in connect
    raise ConnectionError(self._error_message(e))
redis.exceptions.ConnectionError: Error -2 connecting to redis:6379. Name or service not known.
[1;92mINFO [2024-07-08 11:31:35 - idle.py:75] - Stop signal received (SIGINT). Exiting...[0m
[1;92mINFO [2024-07-08 11:32:13 - tcp.py:67] - Using proxy 127.0.0.1[0m
[1;92mINFO [2024-07-08 11:32:13 - connection.py:47] - Connecting...[0m
[1;92mINFO [2024-07-08 11:32:13 - connection.py:54] - Connected! Production DC4 - IPv4[0m
[1;92mINFO [2024-07-08 11:32:13 - session.py:294] - NetworkTask started[0m
[1;92mINFO [2024-07-08 11:32:15 - session.py:139] - Session initialized: Layer 158[0m
[1;92mINFO [2024-07-08 11:32:15 - session.py:140] - Device: CPython 3.10.12 - Pyrogram 2.0.106[0m
[1;92mINFO [2024-07-08 11:32:15 - session.py:141] - System: Linux 5.15.0-101-generic (en)[0m
[1;92mINFO [2024-07-08 11:32:15 - session.py:155] - Session started[0m
[1;92mINFO [2024-07-08 11:32:15 - session.py:272] - PingTask started[0m
[1;92mINFO [2024-07-08 11:32:16 - client.py:698] - [vid-compressbot] [LOAD] MessageHandler("say_hello") in group 0 from "plugins.start"[0m
[1;92mINFO [2024-07-08 11:32:16 - client.py:775] - [vid-compressbot] Successfully loaded 1 plugin from "plugins"[0m
[1;92mINFO [2024-07-08 11:32:16 - dispatcher.py:154] - Started 8 HandlerTasks[0m
[1;91mERROR [2024-07-08 11:32:20 - dispatcher.py:253] - '>' not supported between instances of 'float' and 'str'[0m
Traceback (most recent call last):
  File "/home/freeguy/.local/lib/python3.10/site-packages/pyrogram/dispatcher.py", line 240, in handler_worker
    await handler.callback(self.client, *args)
  File "/home/freeguy/Desktop/project/vid-compress/app/plugins/start.py", line 9, in say_hello
    data = con.setting
  File "/home/freeguy/Desktop/project/vid-compress/app/utils/connection.py", line 34, in setting
    if last_request_time is None or current_time - float(last_request_time) > CACHE_TTL:
TypeError: '>' not supported between instances of 'float' and 'str'
[1;92mINFO [2024-07-08 11:32:38 - idle.py:75] - Stop signal received (SIGINT). Exiting...[0m
[1;92mINFO [2024-07-08 11:32:40 - tcp.py:67] - Using proxy 127.0.0.1[0m
[1;92mINFO [2024-07-08 11:32:40 - connection.py:47] - Connecting...[0m
[1;92mINFO [2024-07-08 11:32:40 - connection.py:54] - Connected! Production DC4 - IPv4[0m
[1;92mINFO [2024-07-08 11:32:40 - session.py:294] - NetworkTask started[0m
[1;92mINFO [2024-07-08 11:32:40 - session.py:139] - Session initialized: Layer 158[0m
[1;92mINFO [2024-07-08 11:32:40 - session.py:140] - Device: CPython 3.10.12 - Pyrogram 2.0.106[0m
[1;92mINFO [2024-07-08 11:32:40 - session.py:141] - System: Linux 5.15.0-101-generic (en)[0m
[1;92mINFO [2024-07-08 11:32:40 - session.py:155] - Session started[0m
[1;92mINFO [2024-07-08 11:32:40 - session.py:272] - PingTask started[0m
[1;92mINFO [2024-07-08 11:32:41 - client.py:698] - [vid-compressbot] [LOAD] MessageHandler("say_hello") in group 0 from "plugins.start"[0m
[1;92mINFO [2024-07-08 11:32:41 - client.py:775] - [vid-compressbot] Successfully loaded 1 plugin from "plugins"[0m
[1;92mINFO [2024-07-08 11:32:41 - dispatcher.py:154] - Started 8 HandlerTasks[0m
[1;92mINFO [2024-07-08 11:32:47 - idle.py:75] - Stop signal received (SIGINT). Exiting...[0m
[1;92mINFO [2024-07-08 11:33:21 - tcp.py:67] - Using proxy 127.0.0.1[0m
[1;92mINFO [2024-07-08 11:33:21 - connection.py:47] - Connecting...[0m
[1;92mINFO [2024-07-08 11:33:21 - connection.py:54] - Connected! Production DC4 - IPv4[0m
[1;92mINFO [2024-07-08 11:33:21 - session.py:294] - NetworkTask started[0m
[1;92mINFO [2024-07-08 11:33:21 - session.py:139] - Session initialized: Layer 158[0m
[1;92mINFO [2024-07-08 11:33:21 - session.py:140] - Device: CPython 3.10.12 - Pyrogram 2.0.106[0m
[1;92mINFO [2024-07-08 11:33:21 - session.py:141] - System: Linux 5.15.0-101-generic (en)[0m
[1;92mINFO [2024-07-08 11:33:21 - session.py:155] - Session started[0m
[1;92mINFO [2024-07-08 11:33:21 - session.py:272] - PingTask started[0m
[1;92mINFO [2024-07-08 11:33:22 - client.py:698] - [vid-compressbot] [LOAD] MessageHandler("say_hello") in group 0 from "plugins.start"[0m
[1;92mINFO [2024-07-08 11:33:22 - client.py:775] - [vid-compressbot] Successfully loaded 1 plugin from "plugins"[0m
[1;92mINFO [2024-07-08 11:33:22 - dispatcher.py:154] - Started 8 HandlerTasks[0m
[1;92mINFO [2024-07-08 11:35:01 - idle.py:75] - Stop signal received (SIGINT). Exiting...[0m
[1;92mINFO [2024-07-08 11:36:20 - tcp.py:67] - Using proxy 127.0.0.1[0m
[1;92mINFO [2024-07-08 11:36:20 - connection.py:47] - Connecting...[0m
[1;92mINFO [2024-07-08 11:36:20 - connection.py:54] - Connected! Production DC4 - IPv4[0m
[1;92mINFO [2024-07-08 11:36:20 - session.py:294] - NetworkTask started[0m
[1;92mINFO [2024-07-08 11:36:20 - session.py:139] - Session initialized: Layer 158[0m
[1;92mINFO [2024-07-08 11:36:20 - session.py:140] - Device: CPython 3.10.12 - Pyrogram 2.0.106[0m
[1;92mINFO [2024-07-08 11:36:20 - session.py:141] - System: Linux 5.15.0-101-generic (en)[0m
[1;92mINFO [2024-07-08 11:36:20 - session.py:155] - Session started[0m
[1;92mINFO [2024-07-08 11:36:20 - session.py:272] - PingTask started[0m
[1;92mINFO [2024-07-08 11:36:21 - client.py:698] - [vid-compressbot] [LOAD] MessageHandler("say_hello") in group 0 from "plugins.start"[0m
[1;92mINFO [2024-07-08 11:36:21 - client.py:775] - [vid-compressbot] Successfully loaded 1 plugin from "plugins"[0m
[1;92mINFO [2024-07-08 11:36:21 - dispatcher.py:154] - Started 8 HandlerTasks[0m
[1;92mINFO [2024-07-08 11:47:44 - idle.py:75] - Stop signal received (SIGINT). Exiting...[0m
[1;92mINFO [2024-07-08 11:47:46 - tcp.py:67] - Using proxy 127.0.0.1[0m
[1;92mINFO [2024-07-08 11:47:46 - connection.py:47] - Connecting...[0m
[1;92mINFO [2024-07-08 11:47:46 - connection.py:54] - Connected! Production DC4 - IPv4[0m
[1;92mINFO [2024-07-08 11:47:46 - session.py:294] - NetworkTask started[0m
[1;92mINFO [2024-07-08 11:47:47 - session.py:139] - Session initialized: Layer 158[0m
[1;92mINFO [2024-07-08 11:47:47 - session.py:140] - Device: CPython 3.10.12 - Pyrogram 2.0.106[0m
[1;92mINFO [2024-07-08 11:47:47 - session.py:141] - System: Linux 5.15.0-101-generic (en)[0m
[1;92mINFO [2024-07-08 11:47:47 - session.py:155] - Session started[0m
[1;92mINFO [2024-07-08 11:47:47 - session.py:272] - PingTask started[0m
[1;92mINFO [2024-07-08 11:48:01 - tcp.py:67] - Using proxy 127.0.0.1[0m
[1;92mINFO [2024-07-08 11:48:01 - connection.py:47] - Connecting...[0m
[1;92mINFO [2024-07-08 11:48:02 - connection.py:54] - Connected! Production DC4 - IPv4[0m
[1;92mINFO [2024-07-08 11:48:02 - session.py:294] - NetworkTask started[0m
[1;92mINFO [2024-07-08 11:48:02 - session.py:139] - Session initialized: Layer 158[0m
[1;92mINFO [2024-07-08 11:48:02 - session.py:140] - Device: CPython 3.10.12 - Pyrogram 2.0.106[0m
[1;92mINFO [2024-07-08 11:48:02 - session.py:141] - System: Linux 5.15.0-101-generic (en)[0m
[1;92mINFO [2024-07-08 11:48:02 - session.py:155] - Session started[0m
[1;92mINFO [2024-07-08 11:48:02 - session.py:272] - PingTask started[0m
[1;92mINFO [2024-07-08 11:48:03 - client.py:698] - [vid-compressbot] [LOAD] MessageHandler("say_hello") in group 0 from "plugins.start"[0m
[1;92mINFO [2024-07-08 11:48:03 - client.py:775] - [vid-compressbot] Successfully loaded 1 plugin from "plugins"[0m
[1;92mINFO [2024-07-08 11:48:03 - dispatcher.py:154] - Started 8 HandlerTasks[0m
[1;92mINFO [2024-07-08 11:50:21 - idle.py:75] - Stop signal received (SIGINT). Exiting...[0m
[1;92mINFO [2024-07-08 11:50:22 - tcp.py:67] - Using proxy 127.0.0.1[0m
[1;92mINFO [2024-07-08 11:50:22 - connection.py:47] - Connecting...[0m
[1;92mINFO [2024-07-08 11:50:22 - connection.py:54] - Connected! Production DC4 - IPv4[0m
[1;92mINFO [2024-07-08 11:50:22 - session.py:294] - NetworkTask started[0m
[1;92mINFO [2024-07-08 11:50:23 - session.py:139] - Session initialized: Layer 158[0m
[1;92mINFO [2024-07-08 11:50:23 - session.py:140] - Device: CPython 3.10.12 - Pyrogram 2.0.106[0m
[1;92mINFO [2024-07-08 11:50:23 - session.py:141] - System: Linux 5.15.0-101-generic (en)[0m
[1;92mINFO [2024-07-08 11:50:23 - session.py:155] - Session started[0m
[1;92mINFO [2024-07-08 11:50:23 - session.py:272] - PingTask started[0m
[1;92mINFO [2024-07-08 11:50:24 - client.py:698] - [vid-compressbot] [LOAD] MessageHandler("say_hello") in group 0 from "plugins.start"[0m
[1;92mINFO [2024-07-08 11:50:24 - client.py:775] - [vid-compressbot] Successfully loaded 1 plugin from "plugins"[0m
[1;92mINFO [2024-07-08 11:50:24 - dispatcher.py:154] - Started 8 HandlerTasks[0m
[1;92mINFO [2024-07-08 11:50:58 - idle.py:75] - Stop signal received (SIGINT). Exiting...[0m
[1;92mINFO [2024-07-08 11:50:59 - tcp.py:67] - Using proxy 127.0.0.1[0m
[1;92mINFO [2024-07-08 11:50:59 - connection.py:47] - Connecting...[0m
[1;92mINFO [2024-07-08 11:51:00 - connection.py:54] - Connected! Production DC4 - IPv4[0m
[1;92mINFO [2024-07-08 11:51:00 - session.py:294] - NetworkTask started[0m
[1;92mINFO [2024-07-08 11:51:00 - session.py:139] - Session initialized: Layer 158[0m
[1;92mINFO [2024-07-08 11:51:00 - session.py:140] - Device: CPython 3.10.12 - Pyrogram 2.0.106[0m
[1;92mINFO [2024-07-08 11:51:00 - session.py:141] - System: Linux 5.15.0-101-generic (en)[0m
[1;92mINFO [2024-07-08 11:51:00 - session.py:155] - Session started[0m
[1;92mINFO [2024-07-08 11:51:00 - session.py:272] - PingTask started[0m
[1;92mINFO [2024-07-08 11:51:01 - client.py:698] - [vid-compressbot] [LOAD] MessageHandler("say_hello") in group 0 from "plugins.start"[0m
[1;92mINFO [2024-07-08 11:51:01 - client.py:775] - [vid-compressbot] Successfully loaded 1 plugin from "plugins"[0m
[1;92mINFO [2024-07-08 11:51:01 - dispatcher.py:154] - Started 8 HandlerTasks[0m
[1;92mINFO [2024-07-08 11:51:29 - idle.py:75] - Stop signal received (SIGINT). Exiting...[0m
[1;92mINFO [2024-07-08 11:51:30 - tcp.py:67] - Using proxy 127.0.0.1[0m
[1;92mINFO [2024-07-08 11:51:30 - connection.py:47] - Connecting...[0m
[1;92mINFO [2024-07-08 11:51:31 - connection.py:54] - Connected! Production DC4 - IPv4[0m
[1;92mINFO [2024-07-08 11:51:31 - session.py:294] - NetworkTask started[0m
[1;92mINFO [2024-07-08 11:51:31 - session.py:139] - Session initialized: Layer 158[0m
[1;92mINFO [2024-07-08 11:51:31 - session.py:140] - Device: CPython 3.10.12 - Pyrogram 2.0.106[0m
[1;92mINFO [2024-07-08 11:51:31 - session.py:141] - System: Linux 5.15.0-101-generic (en)[0m
[1;92mINFO [2024-07-08 11:51:31 - session.py:155] - Session started[0m
[1;92mINFO [2024-07-08 11:51:31 - session.py:272] - PingTask started[0m
[1;92mINFO [2024-07-08 11:51:32 - client.py:698] - [vid-compressbot] [LOAD] MessageHandler("say_hello") in group 0 from "plugins.start"[0m
[1;92mINFO [2024-07-08 11:51:32 - client.py:775] - [vid-compressbot] Successfully loaded 1 plugin from "plugins"[0m
[1;92mINFO [2024-07-08 11:51:32 - dispatcher.py:154] - Started 8 HandlerTasks[0m
[1;92mINFO [2024-07-08 11:51:53 - idle.py:75] - Stop signal received (SIGINT). Exiting...[0m
[1;92mINFO [2024-07-08 11:52:32 - tcp.py:67] - Using proxy 127.0.0.1[0m
[1;92mINFO [2024-07-08 11:52:32 - connection.py:47] - Connecting...[0m
[1;92mINFO [2024-07-08 11:52:32 - connection.py:54] - Connected! Production DC4 - IPv4[0m
[1;92mINFO [2024-07-08 11:52:32 - session.py:294] - NetworkTask started[0m
[1;92mINFO [2024-07-08 11:52:32 - session.py:139] - Session initialized: Layer 158[0m
[1;92mINFO [2024-07-08 11:52:32 - session.py:140] - Device: CPython 3.10.12 - Pyrogram 2.0.106[0m
[1;92mINFO [2024-07-08 11:52:32 - session.py:141] - System: Linux 5.15.0-101-generic (en)[0m
[1;92mINFO [2024-07-08 11:52:32 - session.py:155] - Session started[0m
[1;92mINFO [2024-07-08 11:52:32 - session.py:272] - PingTask started[0m
[1;92mINFO [2024-07-08 11:52:33 - client.py:698] - [vid-compressbot] [LOAD] MessageHandler("user_not_join") in group 0 from "plugins.user_entry"[0m
[1;92mINFO [2024-07-08 11:52:33 - client.py:775] - [vid-compressbot] Successfully loaded 1 plugin from "plugins"[0m
[1;92mINFO [2024-07-08 11:52:33 - dispatcher.py:154] - Started 8 HandlerTasks[0m
[1;92mINFO [2024-07-08 11:53:14 - idle.py:75] - Stop signal received (SIGINT). Exiting...[0m
[1;92mINFO [2024-07-08 11:53:17 - tcp.py:67] - Using proxy 127.0.0.1[0m
[1;92mINFO [2024-07-08 11:53:17 - connection.py:47] - Connecting...[0m
[1;92mINFO [2024-07-08 11:53:17 - connection.py:54] - Connected! Production DC4 - IPv4[0m
[1;92mINFO [2024-07-08 11:53:17 - session.py:294] - NetworkTask started[0m
[1;92mINFO [2024-07-08 11:53:18 - session.py:139] - Session initialized: Layer 158[0m
[1;92mINFO [2024-07-08 11:53:18 - session.py:140] - Device: CPython 3.10.12 - Pyrogram 2.0.106[0m
[1;92mINFO [2024-07-08 11:53:18 - session.py:141] - System: Linux 5.15.0-101-generic (en)[0m
[1;92mINFO [2024-07-08 11:53:18 - session.py:155] - Session started[0m
[1;92mINFO [2024-07-08 11:53:18 - session.py:272] - PingTask started[0m
[1;92mINFO [2024-07-08 11:53:18 - client.py:698] - [vid-compressbot] [LOAD] MessageHandler("user_not_join") in group 0 from "plugins.user_entry"[0m
[1;92mINFO [2024-07-08 11:53:18 - client.py:775] - [vid-compressbot] Successfully loaded 1 plugin from "plugins"[0m
[1;92mINFO [2024-07-08 11:53:18 - dispatcher.py:154] - Started 8 HandlerTasks[0m
[1;92mINFO [2024-07-08 11:54:19 - idle.py:75] - Stop signal received (SIGINT). Exiting...[0m
[1;92mINFO [2024-07-08 11:54:20 - tcp.py:67] - Using proxy 127.0.0.1[0m
[1;92mINFO [2024-07-08 11:54:20 - connection.py:47] - Connecting...[0m
[1;92mINFO [2024-07-08 11:54:21 - connection.py:54] - Connected! Production DC4 - IPv4[0m
[1;92mINFO [2024-07-08 11:54:21 - session.py:294] - NetworkTask started[0m
[1;92mINFO [2024-07-08 11:54:21 - session.py:139] - Session initialized: Layer 158[0m
[1;92mINFO [2024-07-08 11:54:21 - session.py:140] - Device: CPython 3.10.12 - Pyrogram 2.0.106[0m
[1;92mINFO [2024-07-08 11:54:21 - session.py:141] - System: Linux 5.15.0-101-generic (en)[0m
[1;92mINFO [2024-07-08 11:54:21 - session.py:155] - Session started[0m
[1;92mINFO [2024-07-08 11:54:21 - session.py:272] - PingTask started[0m
[1;92mINFO [2024-07-08 11:54:22 - client.py:698] - [vid-compressbot] [LOAD] MessageHandler("user_not_join") in group 0 from "plugins.user_entry"[0m
[1;92mINFO [2024-07-08 11:54:22 - client.py:775] - [vid-compressbot] Successfully loaded 1 plugin from "plugins"[0m
[1;92mINFO [2024-07-08 11:54:22 - dispatcher.py:154] - Started 8 HandlerTasks[0m
[1;92mINFO [2024-07-08 12:00:18 - idle.py:75] - Stop signal received (SIGINT). Exiting...[0m
[1;92mINFO [2024-07-08 12:00:21 - tcp.py:67] - Using proxy 127.0.0.1[0m
[1;92mINFO [2024-07-08 12:00:21 - connection.py:47] - Connecting...[0m
[1;92mINFO [2024-07-08 12:00:21 - connection.py:54] - Connected! Production DC4 - IPv4[0m
[1;92mINFO [2024-07-08 12:00:21 - session.py:294] - NetworkTask started[0m
[1;92mINFO [2024-07-08 12:00:22 - session.py:139] - Session initialized: Layer 158[0m
[1;92mINFO [2024-07-08 12:00:22 - session.py:140] - Device: CPython 3.10.12 - Pyrogram 2.0.106[0m
[1;92mINFO [2024-07-08 12:00:22 - session.py:141] - System: Linux 5.15.0-101-generic (en)[0m
[1;92mINFO [2024-07-08 12:00:22 - session.py:155] - Session started[0m
[1;92mINFO [2024-07-08 12:00:22 - session.py:272] - PingTask started[0m
[1;92mINFO [2024-07-08 12:00:22 - client.py:698] - [vid-compressbot] [LOAD] MessageHandler("user_not_join") in group 0 from "plugins.user_entry"[0m
[1;92mINFO [2024-07-08 12:00:22 - client.py:775] - [vid-compressbot] Successfully loaded 1 plugin from "plugins"[0m
[1;92mINFO [2024-07-08 12:00:22 - dispatcher.py:154] - Started 8 HandlerTasks[0m
[1;92mINFO [2024-07-08 12:00:39 - idle.py:75] - Stop signal received (SIGINT). Exiting...[0m
[1;92mINFO [2024-07-08 12:00:42 - tcp.py:67] - Using proxy 127.0.0.1[0m
[1;92mINFO [2024-07-08 12:00:42 - connection.py:47] - Connecting...[0m
[1;92mINFO [2024-07-08 12:00:42 - connection.py:54] - Connected! Production DC4 - IPv4[0m
[1;92mINFO [2024-07-08 12:00:42 - session.py:294] - NetworkTask started[0m
[1;92mINFO [2024-07-08 12:00:43 - session.py:139] - Session initialized: Layer 158[0m
[1;92mINFO [2024-07-08 12:00:43 - session.py:140] - Device: CPython 3.10.12 - Pyrogram 2.0.106[0m
[1;92mINFO [2024-07-08 12:00:43 - session.py:141] - System: Linux 5.15.0-101-generic (en)[0m
[1;92mINFO [2024-07-08 12:00:43 - session.py:155] - Session started[0m
[1;92mINFO [2024-07-08 12:00:43 - session.py:272] - PingTask started[0m
[1;92mINFO [2024-07-08 12:00:43 - client.py:698] - [vid-compressbot] [LOAD] MessageHandler("user_not_join") in group 0 from "plugins.user_entry"[0m
[1;92mINFO [2024-07-08 12:00:43 - client.py:775] - [vid-compressbot] Successfully loaded 1 plugin from "plugins"[0m
[1;92mINFO [2024-07-08 12:00:43 - dispatcher.py:154] - Started 8 HandlerTasks[0m
[1;92mINFO [2024-07-08 12:00:55 - idle.py:75] - Stop signal received (SIGINT). Exiting...[0m
[1;92mINFO [2024-07-08 14:10:31 - auth.py:82] - Start creating a new auth key on DC2[0m
[1;92mINFO [2024-07-08 14:10:31 - tcp.py:67] - Using proxy 127.0.0.1[0m
[1;92mINFO [2024-07-08 14:10:31 - connection.py:47] - Connecting...[0m
[1;92mINFO [2024-07-08 14:10:31 - connection.py:54] - Connected! Production DC2 - IPv4[0m
[1;92mINFO [2024-07-08 14:10:32 - auth.py:267] - Done auth key exchange: DhGenOk[0m
[1;92mINFO [2024-07-08 14:10:32 - connection.py:66] - Disconnected[0m
[1;92mINFO [2024-07-08 14:10:32 - tcp.py:67] - Using proxy 127.0.0.1[0m
[1;92mINFO [2024-07-08 14:10:32 - connection.py:47] - Connecting...[0m
[1;92mINFO [2024-07-08 14:10:33 - connection.py:54] - Connected! Production DC2 - IPv4[0m
[1;92mINFO [2024-07-08 14:10:33 - session.py:294] - NetworkTask started[0m
[1;92mINFO [2024-07-08 14:10:33 - session.py:139] - Session initialized: Layer 158[0m
[1;92mINFO [2024-07-08 14:10:33 - session.py:140] - Device: CPython 3.10.12 - Pyrogram 2.0.106[0m
[1;92mINFO [2024-07-08 14:10:33 - session.py:141] - System: Linux 5.15.0-101-generic (en)[0m
[1;92mINFO [2024-07-08 14:10:33 - session.py:155] - Session started[0m
[1;92mINFO [2024-07-08 14:10:33 - session.py:272] - PingTask started[0m
[1;92mINFO [2024-07-08 14:10:34 - session.py:291] - PingTask stopped[0m
[1;92mINFO [2024-07-08 14:10:34 - connection.py:66] - Disconnected[0m
[1;92mINFO [2024-07-08 14:10:34 - session.py:315] - NetworkTask stopped[0m
[1;92mINFO [2024-07-08 14:10:34 - session.py:180] - Session stopped[0m
[1;92mINFO [2024-07-08 14:10:34 - auth.py:82] - Start creating a new auth key on DC4[0m
[1;92mINFO [2024-07-08 14:10:34 - tcp.py:67] - Using proxy 127.0.0.1[0m
[1;92mINFO [2024-07-08 14:10:34 - connection.py:47] - Connecting...[0m
[1;92mINFO [2024-07-08 14:10:34 - connection.py:54] - Connected! Production DC4 - IPv4[0m
[1;92mINFO [2024-07-08 14:10:36 - auth.py:267] - Done auth key exchange: DhGenOk[0m
[1;92mINFO [2024-07-08 14:10:36 - connection.py:66] - Disconnected[0m
[1;92mINFO [2024-07-08 14:10:36 - tcp.py:67] - Using proxy 127.0.0.1[0m
[1;92mINFO [2024-07-08 14:10:36 - connection.py:47] - Connecting...[0m
[1;92mINFO [2024-07-08 14:10:36 - connection.py:54] - Connected! Production DC4 - IPv4[0m
[1;92mINFO [2024-07-08 14:10:36 - session.py:294] - NetworkTask started[0m
[1;92mINFO [2024-07-08 14:10:37 - session.py:139] - Session initialized: Layer 158[0m
[1;92mINFO [2024-07-08 14:10:37 - session.py:140] - Device: CPython 3.10.12 - Pyrogram 2.0.106[0m
[1;92mINFO [2024-07-08 14:10:37 - session.py:141] - System: Linux 5.15.0-101-generic (en)[0m
[1;92mINFO [2024-07-08 14:10:37 - session.py:155] - Session started[0m
[1;92mINFO [2024-07-08 14:10:37 - session.py:272] - PingTask started[0m
[1;92mINFO [2024-07-08 14:10:37 - client.py:698] - [vid-compressbot] [LOAD] MessageHandler("user_not_join") in group 0 from "plugins.user_entry"[0m
[1;92mINFO [2024-07-08 14:10:37 - client.py:775] - [vid-compressbot] Successfully loaded 1 plugin from "plugins"[0m
[1;92mINFO [2024-07-08 14:10:37 - dispatcher.py:154] - Started 8 HandlerTasks[0m
[1;91mERROR [2024-07-08 14:10:48 - dispatcher.py:253] - 'NoneType' object has no attribute 'lang'[0m
Traceback (most recent call last):
  File "/home/freeguy/.local/lib/python3.10/site-packages/pyrogram/dispatcher.py", line 240, in handler_worker
    await handler.callback(self.client, *args)
  File "/home/freeguy/Desktop/project/vid-compress/app/plugins/user_entry.py", line 19, in user_not_join
    user_lang = user.lang
AttributeError: 'NoneType' object has no attribute 'lang'
[1;92mINFO [2024-07-08 14:11:05 - idle.py:75] - Stop signal received (SIGINT). Exiting...[0m
[1;92mINFO [2024-07-08 14:11:07 - tcp.py:67] - Using proxy 127.0.0.1[0m
[1;92mINFO [2024-07-08 14:11:07 - connection.py:47] - Connecting...[0m
[1;92mINFO [2024-07-08 14:11:07 - connection.py:54] - Connected! Production DC4 - IPv4[0m
[1;92mINFO [2024-07-08 14:11:07 - session.py:294] - NetworkTask started[0m
[1;92mINFO [2024-07-08 14:11:07 - session.py:139] - Session initialized: Layer 158[0m
[1;92mINFO [2024-07-08 14:11:07 - session.py:140] - Device: CPython 3.10.12 - Pyrogram 2.0.106[0m
[1;92mINFO [2024-07-08 14:11:07 - session.py:141] - System: Linux 5.15.0-101-generic (en)[0m
[1;92mINFO [2024-07-08 14:11:07 - session.py:155] - Session started[0m
[1;92mINFO [2024-07-08 14:11:07 - session.py:272] - PingTask started[0m
[1;92mINFO [2024-07-08 14:11:08 - client.py:698] - [vid-compressbot] [LOAD] MessageHandler("user_not_join") in group 0 from "plugins.user_entry"[0m
[1;92mINFO [2024-07-08 14:11:08 - client.py:775] - [vid-compressbot] Successfully loaded 1 plugin from "plugins"[0m
[1;92mINFO [2024-07-08 14:11:08 - dispatcher.py:154] - Started 8 HandlerTasks[0m
[1;92mINFO [2024-07-08 14:11:33 - idle.py:75] - Stop signal received (SIGINT). Exiting...[0m
[1;92mINFO [2024-07-08 14:11:34 - tcp.py:67] - Using proxy 127.0.0.1[0m
[1;92mINFO [2024-07-08 14:11:34 - connection.py:47] - Connecting...[0m
[1;92mINFO [2024-07-08 14:11:34 - connection.py:54] - Connected! Production DC4 - IPv4[0m
[1;92mINFO [2024-07-08 14:11:34 - session.py:294] - NetworkTask started[0m
[1;92mINFO [2024-07-08 14:11:35 - session.py:139] - Session initialized: Layer 158[0m
[1;92mINFO [2024-07-08 14:11:35 - session.py:140] - Device: CPython 3.10.12 - Pyrogram 2.0.106[0m
[1;92mINFO [2024-07-08 14:11:35 - session.py:141] - System: Linux 5.15.0-101-generic (en)[0m
[1;92mINFO [2024-07-08 14:11:35 - session.py:155] - Session started[0m
[1;92mINFO [2024-07-08 14:11:35 - session.py:272] - PingTask started[0m
[1;92mINFO [2024-07-08 14:11:35 - client.py:698] - [vid-compressbot] [LOAD] MessageHandler("user_not_join") in group 0 from "plugins.user_entry"[0m
[1;92mINFO [2024-07-08 14:11:35 - client.py:775] - [vid-compressbot] Successfully loaded 1 plugin from "plugins"[0m
[1;92mINFO [2024-07-08 14:11:35 - dispatcher.py:154] - Started 8 HandlerTasks[0m
[1;92mINFO [2024-07-08 14:11:41 - idle.py:75] - Stop signal received (SIGINT). Exiting...[0m
[1;92mINFO [2024-07-08 14:12:49 - tcp.py:67] - Using proxy 127.0.0.1[0m
[1;92mINFO [2024-07-08 14:12:49 - connection.py:47] - Connecting...[0m
[1;92mINFO [2024-07-08 14:12:50 - connection.py:54] - Connected! Production DC4 - IPv4[0m
[1;92mINFO [2024-07-08 14:12:50 - session.py:294] - NetworkTask started[0m
[1;92mINFO [2024-07-08 14:12:50 - session.py:139] - Session initialized: Layer 158[0m
[1;92mINFO [2024-07-08 14:12:50 - session.py:140] - Device: CPython 3.10.12 - Pyrogram 2.0.106[0m
[1;92mINFO [2024-07-08 14:12:50 - session.py:141] - System: Linux 5.15.0-101-generic (en)[0m
[1;92mINFO [2024-07-08 14:12:50 - session.py:155] - Session started[0m
[1;92mINFO [2024-07-08 14:12:50 - session.py:272] - PingTask started[0m
[1;92mINFO [2024-07-08 14:12:51 - client.py:698] - [vid-compressbot] [LOAD] MessageHandler("user_not_join") in group 0 from "plugins.user_entry"[0m
[1;92mINFO [2024-07-08 14:12:51 - client.py:775] - [vid-compressbot] Successfully loaded 1 plugin from "plugins"[0m
[1;92mINFO [2024-07-08 14:12:51 - dispatcher.py:154] - Started 8 HandlerTasks[0m
[1;92mINFO [2024-07-08 14:13:55 - idle.py:75] - Stop signal received (SIGINT). Exiting...[0m
[1;92mINFO [2024-07-08 14:13:56 - tcp.py:67] - Using proxy 127.0.0.1[0m
[1;92mINFO [2024-07-08 14:13:56 - connection.py:47] - Connecting...[0m
[1;92mINFO [2024-07-08 14:13:56 - connection.py:54] - Connected! Production DC4 - IPv4[0m
[1;92mINFO [2024-07-08 14:13:56 - session.py:294] - NetworkTask started[0m
[1;92mINFO [2024-07-08 14:13:57 - session.py:139] - Session initialized: Layer 158[0m
[1;92mINFO [2024-07-08 14:13:57 - session.py:140] - Device: CPython 3.10.12 - Pyrogram 2.0.106[0m
[1;92mINFO [2024-07-08 14:13:57 - session.py:141] - System: Linux 5.15.0-101-generic (en)[0m
[1;92mINFO [2024-07-08 14:13:57 - session.py:155] - Session started[0m
[1;92mINFO [2024-07-08 14:13:57 - session.py:272] - PingTask started[0m
[1;92mINFO [2024-07-08 14:13:57 - client.py:698] - [vid-compressbot] [LOAD] MessageHandler("user_not_join") in group 0 from "plugins.user_entry"[0m
[1;92mINFO [2024-07-08 14:13:57 - client.py:775] - [vid-compressbot] Successfully loaded 1 plugin from "plugins"[0m
[1;92mINFO [2024-07-08 14:13:57 - dispatcher.py:154] - Started 8 HandlerTasks[0m
[1;92mINFO [2024-07-08 14:17:10 - idle.py:75] - Stop signal received (SIGINT). Exiting...[0m
[1;92mINFO [2024-07-08 14:17:12 - tcp.py:67] - Using proxy 127.0.0.1[0m
[1;92mINFO [2024-07-08 14:17:12 - connection.py:47] - Connecting...[0m
[1;92mINFO [2024-07-08 14:17:12 - connection.py:54] - Connected! Production DC4 - IPv4[0m
[1;92mINFO [2024-07-08 14:17:12 - session.py:294] - NetworkTask started[0m
[1;92mINFO [2024-07-08 14:17:13 - session.py:139] - Session initialized: Layer 158[0m
[1;92mINFO [2024-07-08 14:17:13 - session.py:140] - Device: CPython 3.10.12 - Pyrogram 2.0.106[0m
[1;92mINFO [2024-07-08 14:17:13 - session.py:141] - System: Linux 5.15.0-101-generic (en)[0m
[1;92mINFO [2024-07-08 14:17:13 - session.py:155] - Session started[0m
[1;92mINFO [2024-07-08 14:17:13 - session.py:272] - PingTask started[0m
[1;92mINFO [2024-07-08 14:17:13 - client.py:698] - [vid-compressbot] [LOAD] MessageHandler("user_not_join") in group 0 from "plugins.user_entry"[0m
[1;92mINFO [2024-07-08 14:17:13 - client.py:775] - [vid-compressbot] Successfully loaded 1 plugin from "plugins"[0m
[1;92mINFO [2024-07-08 14:17:13 - dispatcher.py:154] - Started 8 HandlerTasks[0m
[1;91mERROR [2024-07-08 14:17:17 - dispatcher.py:253] - 'Response' object is not subscriptable[0m
Traceback (most recent call last):
  File "/home/freeguy/.local/lib/python3.10/site-packages/pyrogram/dispatcher.py", line 240, in handler_worker
    await handler.callback(self.client, *args)
  File "/home/freeguy/Desktop/project/vid-compress/app/plugins/user_entry.py", line 18, in user_not_join
    join_text = setting['join_text_fa'] if user['lang'] == 'fa' else setting['join_text_en']
TypeError: 'Response' object is not subscriptable
[1;91mERROR [2024-07-08 14:17:17 - dispatcher.py:229] - 'NoneType' object has no attribute 'id'[0m
Traceback (most recent call last):
  File "/home/freeguy/.local/lib/python3.10/site-packages/pyrogram/dispatcher.py", line 226, in handler_worker
    if await handler.check(self.client, parsed_update):
  File "/home/freeguy/.local/lib/python3.10/site-packages/pyrogram/handlers/handler.py", line 35, in check
    return await self.filters(client, update)
  File "/home/freeguy/.local/lib/python3.10/site-packages/pyrogram/filters.py", line 66, in __call__
    x = await self.base(client, update)
  File "/home/freeguy/Desktop/project/vid-compress/app/utils/filters.py", line 21, in user_not_join
    con.user(chat_id=msg.from_user.id , full_name=msg.from_user.first_name )
AttributeError: 'NoneType' object has no attribute 'id'
[1;91mERROR [2024-07-08 14:17:46 - dispatcher.py:229] - 'NoneType' object has no attribute 'id'[0m
Traceback (most recent call last):
  File "/home/freeguy/.local/lib/python3.10/site-packages/pyrogram/dispatcher.py", line 226, in handler_worker
    if await handler.check(self.client, parsed_update):
  File "/home/freeguy/.local/lib/python3.10/site-packages/pyrogram/handlers/handler.py", line 35, in check
    return await self.filters(client, update)
  File "/home/freeguy/.local/lib/python3.10/site-packages/pyrogram/filters.py", line 66, in __call__
    x = await self.base(client, update)
  File "/home/freeguy/Desktop/project/vid-compress/app/utils/filters.py", line 21, in user_not_join
    con.user(chat_id=msg.from_user.id , full_name=msg.from_user.first_name )
AttributeError: 'NoneType' object has no attribute 'id'
[1;92mINFO [2024-07-08 14:18:02 - idle.py:75] - Stop signal received (SIGINT). Exiting...[0m
[1;92mINFO [2024-07-08 14:18:08 - tcp.py:67] - Using proxy 127.0.0.1[0m
[1;92mINFO [2024-07-08 14:18:08 - connection.py:47] - Connecting...[0m
[1;92mINFO [2024-07-08 14:18:08 - connection.py:54] - Connected! Production DC4 - IPv4[0m
[1;92mINFO [2024-07-08 14:18:08 - session.py:294] - NetworkTask started[0m
[1;92mINFO [2024-07-08 14:18:09 - session.py:139] - Session initialized: Layer 158[0m
[1;92mINFO [2024-07-08 14:18:09 - session.py:140] - Device: CPython 3.10.12 - Pyrogram 2.0.106[0m
[1;92mINFO [2024-07-08 14:18:09 - session.py:141] - System: Linux 5.15.0-101-generic (en)[0m
[1;92mINFO [2024-07-08 14:18:09 - session.py:155] - Session started[0m
[1;92mINFO [2024-07-08 14:18:09 - session.py:272] - PingTask started[0m
[1;92mINFO [2024-07-08 14:18:09 - client.py:698] - [vid-compressbot] [LOAD] MessageHandler("user_not_join") in group 0 from "plugins.user_entry"[0m
[1;92mINFO [2024-07-08 14:18:09 - client.py:775] - [vid-compressbot] Successfully loaded 1 plugin from "plugins"[0m
[1;92mINFO [2024-07-08 14:18:09 - dispatcher.py:154] - Started 8 HandlerTasks[0m
[1;91mERROR [2024-07-08 14:18:12 - dispatcher.py:253] - 'Response' object is not subscriptable[0m
Traceback (most recent call last):
  File "/home/freeguy/.local/lib/python3.10/site-packages/pyrogram/dispatcher.py", line 240, in handler_worker
    await handler.callback(self.client, *args)
  File "/home/freeguy/Desktop/project/vid-compress/app/plugins/user_entry.py", line 18, in user_not_join
    join_text = setting['join_text_fa'] if user['lang'] == 'fa' else setting['join_text_en']
TypeError: 'Response' object is not subscriptable
[1;92mINFO [2024-07-08 14:18:37 - idle.py:75] - Stop signal received (SIGINT). Exiting...[0m
[1;92mINFO [2024-07-08 14:18:39 - tcp.py:67] - Using proxy 127.0.0.1[0m
[1;92mINFO [2024-07-08 14:18:39 - connection.py:47] - Connecting...[0m
[1;92mINFO [2024-07-08 14:18:39 - connection.py:54] - Connected! Production DC4 - IPv4[0m
[1;92mINFO [2024-07-08 14:18:39 - session.py:294] - NetworkTask started[0m
[1;92mINFO [2024-07-08 14:18:39 - session.py:139] - Session initialized: Layer 158[0m
[1;92mINFO [2024-07-08 14:18:39 - session.py:140] - Device: CPython 3.10.12 - Pyrogram 2.0.106[0m
[1;92mINFO [2024-07-08 14:18:39 - session.py:141] - System: Linux 5.15.0-101-generic (en)[0m
[1;92mINFO [2024-07-08 14:18:39 - session.py:155] - Session started[0m
[1;92mINFO [2024-07-08 14:18:39 - session.py:272] - PingTask started[0m
[1;92mINFO [2024-07-08 14:18:40 - client.py:698] - [vid-compressbot] [LOAD] MessageHandler("user_not_join") in group 0 from "plugins.user_entry"[0m
[1;92mINFO [2024-07-08 14:18:40 - client.py:775] - [vid-compressbot] Successfully loaded 1 plugin from "plugins"[0m
[1;92mINFO [2024-07-08 14:18:40 - dispatcher.py:154] - Started 8 HandlerTasks[0m
[1;91mERROR [2024-07-08 14:20:53 - dispatcher.py:229] - 'NoneType' object has no attribute 'id'[0m
Traceback (most recent call last):
  File "/home/freeguy/.local/lib/python3.10/site-packages/pyrogram/dispatcher.py", line 226, in handler_worker
    if await handler.check(self.client, parsed_update):
  File "/home/freeguy/.local/lib/python3.10/site-packages/pyrogram/handlers/handler.py", line 35, in check
    return await self.filters(client, update)
  File "/home/freeguy/.local/lib/python3.10/site-packages/pyrogram/filters.py", line 66, in __call__
    x = await self.base(client, update)
  File "/home/freeguy/Desktop/project/vid-compress/app/utils/filters.py", line 21, in user_not_join
    con.user(chat_id=msg.from_user.id , full_name=msg.from_user.first_name )
AttributeError: 'NoneType' object has no attribute 'id'
[1;91mERROR [2024-07-08 14:24:06 - dispatcher.py:229] - 'NoneType' object has no attribute 'id'[0m
Traceback (most recent call last):
  File "/home/freeguy/.local/lib/python3.10/site-packages/pyrogram/dispatcher.py", line 226, in handler_worker
    if await handler.check(self.client, parsed_update):
  File "/home/freeguy/.local/lib/python3.10/site-packages/pyrogram/handlers/handler.py", line 35, in check
    return await self.filters(client, update)
  File "/home/freeguy/.local/lib/python3.10/site-packages/pyrogram/filters.py", line 66, in __call__
    x = await self.base(client, update)
  File "/home/freeguy/Desktop/project/vid-compress/app/utils/filters.py", line 21, in user_not_join
    con.user(chat_id=msg.from_user.id , full_name=msg.from_user.first_name )
AttributeError: 'NoneType' object has no attribute 'id'
[1;91mERROR [2024-07-08 14:24:06 - dispatcher.py:229] - 'NoneType' object has no attribute 'id'[0m
Traceback (most recent call last):
  File "/home/freeguy/.local/lib/python3.10/site-packages/pyrogram/dispatcher.py", line 226, in handler_worker
    if await handler.check(self.client, parsed_update):
  File "/home/freeguy/.local/lib/python3.10/site-packages/pyrogram/handlers/handler.py", line 35, in check
    return await self.filters(client, update)
  File "/home/freeguy/.local/lib/python3.10/site-packages/pyrogram/filters.py", line 66, in __call__
    x = await self.base(client, update)
  File "/home/freeguy/Desktop/project/vid-compress/app/utils/filters.py", line 21, in user_not_join
    con.user(chat_id=msg.from_user.id , full_name=msg.from_user.first_name )
AttributeError: 'NoneType' object has no attribute 'id'
[1;91mERROR [2024-07-08 14:24:20 - dispatcher.py:229] - 'NoneType' object has no attribute 'id'[0m
Traceback (most recent call last):
  File "/home/freeguy/.local/lib/python3.10/site-packages/pyrogram/dispatcher.py", line 226, in handler_worker
    if await handler.check(self.client, parsed_update):
  File "/home/freeguy/.local/lib/python3.10/site-packages/pyrogram/handlers/handler.py", line 35, in check
    return await self.filters(client, update)
  File "/home/freeguy/.local/lib/python3.10/site-packages/pyrogram/filters.py", line 66, in __call__
    x = await self.base(client, update)
  File "/home/freeguy/Desktop/project/vid-compress/app/utils/filters.py", line 21, in user_not_join
    con.user(chat_id=msg.from_user.id , full_name=msg.from_user.first_name )
AttributeError: 'NoneType' object has no attribute 'id'
[1;91mERROR [2024-07-08 14:24:30 - dispatcher.py:229] - 'NoneType' object has no attribute 'id'[0m
Traceback (most recent call last):
  File "/home/freeguy/.local/lib/python3.10/site-packages/pyrogram/dispatcher.py", line 226, in handler_worker
    if await handler.check(self.client, parsed_update):
  File "/home/freeguy/.local/lib/python3.10/site-packages/pyrogram/handlers/handler.py", line 35, in check
    return await self.filters(client, update)
  File "/home/freeguy/.local/lib/python3.10/site-packages/pyrogram/filters.py", line 66, in __call__
    x = await self.base(client, update)
  File "/home/freeguy/Desktop/project/vid-compress/app/utils/filters.py", line 21, in user_not_join
    con.user(chat_id=msg.from_user.id , full_name=msg.from_user.first_name )
AttributeError: 'NoneType' object has no attribute 'id'
[1;91mERROR [2024-07-08 14:24:41 - dispatcher.py:229] - 'NoneType' object has no attribute 'id'[0m
Traceback (most recent call last):
  File "/home/freeguy/.local/lib/python3.10/site-packages/pyrogram/dispatcher.py", line 226, in handler_worker
    if await handler.check(self.client, parsed_update):
  File "/home/freeguy/.local/lib/python3.10/site-packages/pyrogram/handlers/handler.py", line 35, in check
    return await self.filters(client, update)
  File "/home/freeguy/.local/lib/python3.10/site-packages/pyrogram/filters.py", line 66, in __call__
    x = await self.base(client, update)
  File "/home/freeguy/Desktop/project/vid-compress/app/utils/filters.py", line 21, in user_not_join
    con.user(chat_id=msg.from_user.id , full_name=msg.from_user.first_name )
AttributeError: 'NoneType' object has no attribute 'id'
[1;91mERROR [2024-07-08 14:24:51 - dispatcher.py:229] - 'NoneType' object has no attribute 'id'[0m
Traceback (most recent call last):
  File "/home/freeguy/.local/lib/python3.10/site-packages/pyrogram/dispatcher.py", line 226, in handler_worker
    if await handler.check(self.client, parsed_update):
  File "/home/freeguy/.local/lib/python3.10/site-packages/pyrogram/handlers/handler.py", line 35, in check
    return await self.filters(client, update)
  File "/home/freeguy/.local/lib/python3.10/site-packages/pyrogram/filters.py", line 66, in __call__
    x = await self.base(client, update)
  File "/home/freeguy/Desktop/project/vid-compress/app/utils/filters.py", line 21, in user_not_join
    con.user(chat_id=msg.from_user.id , full_name=msg.from_user.first_name )
AttributeError: 'NoneType' object has no attribute 'id'
[1;91mERROR [2024-07-08 14:25:00 - dispatcher.py:229] - 'NoneType' object has no attribute 'id'[0m
Traceback (most recent call last):
  File "/home/freeguy/.local/lib/python3.10/site-packages/pyrogram/dispatcher.py", line 226, in handler_worker
    if await handler.check(self.client, parsed_update):
  File "/home/freeguy/.local/lib/python3.10/site-packages/pyrogram/handlers/handler.py", line 35, in check
    return await self.filters(client, update)
  File "/home/freeguy/.local/lib/python3.10/site-packages/pyrogram/filters.py", line 66, in __call__
    x = await self.base(client, update)
  File "/home/freeguy/Desktop/project/vid-compress/app/utils/filters.py", line 21, in user_not_join
    con.user(chat_id=msg.from_user.id , full_name=msg.from_user.first_name )
AttributeError: 'NoneType' object has no attribute 'id'
[1;91mERROR [2024-07-08 14:25:10 - dispatcher.py:229] - 'NoneType' object has no attribute 'id'[0m
Traceback (most recent call last):
  File "/home/freeguy/.local/lib/python3.10/site-packages/pyrogram/dispatcher.py", line 226, in handler_worker
    if await handler.check(self.client, parsed_update):
  File "/home/freeguy/.local/lib/python3.10/site-packages/pyrogram/handlers/handler.py", line 35, in check
    return await self.filters(client, update)
  File "/home/freeguy/.local/lib/python3.10/site-packages/pyrogram/filters.py", line 66, in __call__
    x = await self.base(client, update)
  File "/home/freeguy/Desktop/project/vid-compress/app/utils/filters.py", line 21, in user_not_join
    con.user(chat_id=msg.from_user.id , full_name=msg.from_user.first_name )
AttributeError: 'NoneType' object has no attribute 'id'
[1;92mINFO [2024-07-08 14:25:14 - idle.py:75] - Stop signal received (SIGINT). Exiting...[0m
[1;92mINFO [2024-07-08 14:25:15 - tcp.py:67] - Using proxy 127.0.0.1[0m
[1;92mINFO [2024-07-08 14:25:15 - connection.py:47] - Connecting...[0m
[1;92mINFO [2024-07-08 14:25:16 - connection.py:54] - Connected! Production DC4 - IPv4[0m
[1;92mINFO [2024-07-08 14:25:16 - session.py:294] - NetworkTask started[0m
[1;92mINFO [2024-07-08 14:25:16 - session.py:139] - Session initialized: Layer 158[0m
[1;92mINFO [2024-07-08 14:25:16 - session.py:140] - Device: CPython 3.10.12 - Pyrogram 2.0.106[0m
[1;92mINFO [2024-07-08 14:25:16 - session.py:141] - System: Linux 5.15.0-101-generic (en)[0m
[1;92mINFO [2024-07-08 14:25:16 - session.py:155] - Session started[0m
[1;92mINFO [2024-07-08 14:25:16 - session.py:272] - PingTask started[0m
[1;92mINFO [2024-07-08 14:25:17 - client.py:698] - [vid-compressbot] [LOAD] MessageHandler("user_not_join") in group 0 from "plugins.user_entry"[0m
[1;92mINFO [2024-07-08 14:25:17 - client.py:775] - [vid-compressbot] Successfully loaded 1 plugin from "plugins"[0m
[1;92mINFO [2024-07-08 14:25:17 - dispatcher.py:154] - Started 8 HandlerTasks[0m
[1;91mERROR [2024-07-08 14:25:17 - dispatcher.py:229] - 'NoneType' object has no attribute 'id'[0m
Traceback (most recent call last):
  File "/home/freeguy/.local/lib/python3.10/site-packages/pyrogram/dispatcher.py", line 226, in handler_worker
    if await handler.check(self.client, parsed_update):
  File "/home/freeguy/.local/lib/python3.10/site-packages/pyrogram/handlers/handler.py", line 35, in check
    return await self.filters(client, update)
  File "/home/freeguy/.local/lib/python3.10/site-packages/pyrogram/filters.py", line 66, in __call__
    x = await self.base(client, update)
  File "/home/freeguy/Desktop/project/vid-compress/app/utils/filters.py", line 21, in user_not_join
    con.user(chat_id=msg.from_user.id , full_name=msg.from_user.first_name )
AttributeError: 'NoneType' object has no attribute 'id'
[1;91mERROR [2024-07-08 14:25:19 - dispatcher.py:253] - 'list' object has no attribute 'write'[0m
Traceback (most recent call last):
  File "/home/freeguy/.local/lib/python3.10/site-packages/pyrogram/dispatcher.py", line 240, in handler_worker
    await handler.callback(self.client, *args)
  File "/home/freeguy/Desktop/project/vid-compress/app/plugins/user_entry.py", line 20, in user_not_join
    await bot.send_message(msg.from_user.id, text=join_text, reply_markup = btn.join_channel(lang=user.lang , url=setting.channel_url))
  File "/home/freeguy/.local/lib/python3.10/site-packages/pyrogram/methods/messages/send_message.py", line 134, in send_message
    reply_markup=await reply_markup.write(self) if reply_markup else None,
AttributeError: 'list' object has no attribute 'write'
[1;91mERROR [2024-07-08 14:25:19 - dispatcher.py:229] - 'NoneType' object has no attribute 'id'[0m
Traceback (most recent call last):
  File "/home/freeguy/.local/lib/python3.10/site-packages/pyrogram/dispatcher.py", line 226, in handler_worker
    if await handler.check(self.client, parsed_update):
  File "/home/freeguy/.local/lib/python3.10/site-packages/pyrogram/handlers/handler.py", line 35, in check
    return await self.filters(client, update)
  File "/home/freeguy/.local/lib/python3.10/site-packages/pyrogram/filters.py", line 66, in __call__
    x = await self.base(client, update)
  File "/home/freeguy/Desktop/project/vid-compress/app/utils/filters.py", line 21, in user_not_join
    con.user(chat_id=msg.from_user.id , full_name=msg.from_user.first_name )
AttributeError: 'NoneType' object has no attribute 'id'
[1;92mINFO [2024-07-08 14:25:22 - idle.py:75] - Stop signal received (SIGINT). Exiting...[0m
[1;92mINFO [2024-07-08 14:27:16 - tcp.py:67] - Using proxy 127.0.0.1[0m
[1;92mINFO [2024-07-08 14:27:16 - connection.py:47] - Connecting...[0m
[1;92mINFO [2024-07-08 14:27:17 - connection.py:54] - Connected! Production DC4 - IPv4[0m
[1;92mINFO [2024-07-08 14:27:17 - session.py:294] - NetworkTask started[0m
[1;92mINFO [2024-07-08 14:27:17 - session.py:139] - Session initialized: Layer 158[0m
[1;92mINFO [2024-07-08 14:27:17 - session.py:140] - Device: CPython 3.10.12 - Pyrogram 2.0.106[0m
[1;92mINFO [2024-07-08 14:27:17 - session.py:141] - System: Linux 5.15.0-101-generic (en)[0m
[1;92mINFO [2024-07-08 14:27:17 - session.py:155] - Session started[0m
[1;92mINFO [2024-07-08 14:27:17 - session.py:272] - PingTask started[0m
[1;92mINFO [2024-07-08 14:27:18 - client.py:698] - [vid-compressbot] [LOAD] MessageHandler("user_not_join") in group 0 from "plugins.user_entry"[0m
[1;92mINFO [2024-07-08 14:27:18 - client.py:775] - [vid-compressbot] Successfully loaded 1 plugin from "plugins"[0m
[1;92mINFO [2024-07-08 14:27:18 - dispatcher.py:154] - Started 8 HandlerTasks[0m
[1;91mERROR [2024-07-08 14:27:23 - dispatcher.py:253] - 'list' object has no attribute 'write'[0m
Traceback (most recent call last):
  File "/home/freeguy/.local/lib/python3.10/site-packages/pyrogram/dispatcher.py", line 240, in handler_worker
    await handler.callback(self.client, *args)
  File "/home/freeguy/Desktop/project/vid-compress/app/plugins/user_entry.py", line 20, in user_not_join
    await bot.send_message(msg.from_user.id, text=join_text, reply_markup = btn.join_channel(lang=user.lang , url=setting.channel_url))
  File "/home/freeguy/.local/lib/python3.10/site-packages/pyrogram/methods/messages/send_message.py", line 134, in send_message
    reply_markup=await reply_markup.write(self) if reply_markup else None,
AttributeError: 'list' object has no attribute 'write'
[1;92mINFO [2024-07-08 14:27:56 - idle.py:75] - Stop signal received (SIGINT). Exiting...[0m
[1;92mINFO [2024-07-08 14:27:59 - tcp.py:67] - Using proxy 127.0.0.1[0m
[1;92mINFO [2024-07-08 14:27:59 - connection.py:47] - Connecting...[0m
[1;92mINFO [2024-07-08 14:27:59 - connection.py:54] - Connected! Production DC4 - IPv4[0m
[1;92mINFO [2024-07-08 14:27:59 - session.py:294] - NetworkTask started[0m
[1;92mINFO [2024-07-08 14:27:59 - session.py:139] - Session initialized: Layer 158[0m
[1;92mINFO [2024-07-08 14:27:59 - session.py:140] - Device: CPython 3.10.12 - Pyrogram 2.0.106[0m
[1;92mINFO [2024-07-08 14:27:59 - session.py:141] - System: Linux 5.15.0-101-generic (en)[0m
[1;92mINFO [2024-07-08 14:27:59 - session.py:155] - Session started[0m
[1;92mINFO [2024-07-08 14:27:59 - session.py:272] - PingTask started[0m
[1;92mINFO [2024-07-08 14:28:00 - client.py:698] - [vid-compressbot] [LOAD] MessageHandler("user_not_join") in group 0 from "plugins.user_entry"[0m
[1;92mINFO [2024-07-08 14:28:00 - client.py:775] - [vid-compressbot] Successfully loaded 1 plugin from "plugins"[0m
[1;92mINFO [2024-07-08 14:28:00 - dispatcher.py:154] - Started 8 HandlerTasks[0m
[1;91mERROR [2024-07-08 14:28:02 - dispatcher.py:253] - 'list' object has no attribute 'write'[0m
Traceback (most recent call last):
  File "/home/freeguy/.local/lib/python3.10/site-packages/pyrogram/dispatcher.py", line 240, in handler_worker
    await handler.callback(self.client, *args)
  File "/home/freeguy/Desktop/project/vid-compress/app/plugins/user_entry.py", line 20, in user_not_join
    await bot.send_message(msg.from_user.id, text=join_text, reply_markup = btn.join_channel(lang=user.lang , url=setting.channel_url[0]))
  File "/home/freeguy/.local/lib/python3.10/site-packages/pyrogram/methods/messages/send_message.py", line 134, in send_message
    reply_markup=await reply_markup.write(self) if reply_markup else None,
AttributeError: 'list' object has no attribute 'write'
[1;92mINFO [2024-07-08 14:28:03 - idle.py:75] - Stop signal received (SIGINT). Exiting...[0m
[1;92mINFO [2024-07-08 14:28:20 - tcp.py:67] - Using proxy 127.0.0.1[0m
[1;92mINFO [2024-07-08 14:28:20 - connection.py:47] - Connecting...[0m
[1;92mINFO [2024-07-08 14:28:20 - connection.py:54] - Connected! Production DC4 - IPv4[0m
[1;92mINFO [2024-07-08 14:28:20 - session.py:294] - NetworkTask started[0m
[1;92mINFO [2024-07-08 14:28:21 - session.py:139] - Session initialized: Layer 158[0m
[1;92mINFO [2024-07-08 14:28:21 - session.py:140] - Device: CPython 3.10.12 - Pyrogram 2.0.106[0m
[1;92mINFO [2024-07-08 14:28:21 - session.py:141] - System: Linux 5.15.0-101-generic (en)[0m
[1;92mINFO [2024-07-08 14:28:21 - session.py:155] - Session started[0m
[1;92mINFO [2024-07-08 14:28:21 - session.py:272] - PingTask started[0m
[1;92mINFO [2024-07-08 14:28:21 - client.py:698] - [vid-compressbot] [LOAD] MessageHandler("user_not_join") in group 0 from "plugins.user_entry"[0m
[1;92mINFO [2024-07-08 14:28:21 - client.py:775] - [vid-compressbot] Successfully loaded 1 plugin from "plugins"[0m
[1;92mINFO [2024-07-08 14:28:21 - dispatcher.py:154] - Started 8 HandlerTasks[0m
[1;91mERROR [2024-07-08 14:28:51 - dispatcher.py:229] - 'NoneType' object has no attribute 'id'[0m
Traceback (most recent call last):
  File "/home/freeguy/.local/lib/python3.10/site-packages/pyrogram/dispatcher.py", line 226, in handler_worker
    if await handler.check(self.client, parsed_update):
  File "/home/freeguy/.local/lib/python3.10/site-packages/pyrogram/handlers/handler.py", line 35, in check
    return await self.filters(client, update)
  File "/home/freeguy/.local/lib/python3.10/site-packages/pyrogram/filters.py", line 66, in __call__
    x = await self.base(client, update)
  File "/home/freeguy/Desktop/project/vid-compress/app/utils/filters.py", line 22, in user_not_join
    con.user(chat_id=msg.from_user.id , full_name=msg.from_user.first_name )
AttributeError: 'NoneType' object has no attribute 'id'
[1;92mINFO [2024-07-08 14:29:07 - idle.py:75] - Stop signal received (SIGINT). Exiting...[0m
[1;92mINFO [2024-07-08 14:29:11 - tcp.py:67] - Using proxy 127.0.0.1[0m
[1;92mINFO [2024-07-08 14:29:11 - connection.py:47] - Connecting...[0m
[1;92mINFO [2024-07-08 14:29:11 - connection.py:54] - Connected! Production DC4 - IPv4[0m
[1;92mINFO [2024-07-08 14:29:11 - session.py:294] - NetworkTask started[0m
[1;92mINFO [2024-07-08 14:29:12 - session.py:139] - Session initialized: Layer 158[0m
[1;92mINFO [2024-07-08 14:29:12 - session.py:140] - Device: CPython 3.10.12 - Pyrogram 2.0.106[0m
[1;92mINFO [2024-07-08 14:29:12 - session.py:141] - System: Linux 5.15.0-101-generic (en)[0m
[1;92mINFO [2024-07-08 14:29:12 - session.py:155] - Session started[0m
[1;92mINFO [2024-07-08 14:29:12 - session.py:272] - PingTask started[0m
[1;92mINFO [2024-07-08 14:29:12 - client.py:698] - [vid-compressbot] [LOAD] MessageHandler("user_not_join") in group 0 from "plugins.user_entry"[0m
[1;92mINFO [2024-07-08 14:29:12 - client.py:775] - [vid-compressbot] Successfully loaded 1 plugin from "plugins"[0m
[1;92mINFO [2024-07-08 14:29:12 - dispatcher.py:154] - Started 8 HandlerTasks[0m
[1;92mINFO [2024-07-08 14:29:33 - idle.py:75] - Stop signal received (SIGINT). Exiting...[0m
[1;92mINFO [2024-07-08 14:29:34 - tcp.py:67] - Using proxy 127.0.0.1[0m
[1;92mINFO [2024-07-08 14:29:34 - connection.py:47] - Connecting...[0m
[1;92mINFO [2024-07-08 14:29:34 - connection.py:54] - Connected! Production DC4 - IPv4[0m
[1;92mINFO [2024-07-08 14:29:34 - session.py:294] - NetworkTask started[0m
[1;92mINFO [2024-07-08 14:29:35 - session.py:139] - Session initialized: Layer 158[0m
[1;92mINFO [2024-07-08 14:29:35 - session.py:140] - Device: CPython 3.10.12 - Pyrogram 2.0.106[0m
[1;92mINFO [2024-07-08 14:29:35 - session.py:141] - System: Linux 5.15.0-101-generic (en)[0m
[1;92mINFO [2024-07-08 14:29:35 - session.py:155] - Session started[0m
[1;92mINFO [2024-07-08 14:29:35 - session.py:272] - PingTask started[0m
[1;92mINFO [2024-07-08 14:29:35 - client.py:698] - [vid-compressbot] [LOAD] MessageHandler("user_not_join") in group 0 from "plugins.user_entry"[0m
[1;92mINFO [2024-07-08 14:29:35 - client.py:775] - [vid-compressbot] Successfully loaded 1 plugin from "plugins"[0m
[1;92mINFO [2024-07-08 14:29:35 - dispatcher.py:154] - Started 8 HandlerTasks[0m
[1;92mINFO [2024-07-08 14:29:42 - idle.py:75] - Stop signal received (SIGINT). Exiting...[0m
[1;92mINFO [2024-07-08 14:30:08 - tcp.py:67] - Using proxy 127.0.0.1[0m
[1;92mINFO [2024-07-08 14:30:08 - connection.py:47] - Connecting...[0m
[1;92mINFO [2024-07-08 14:30:08 - connection.py:54] - Connected! Production DC4 - IPv4[0m
[1;92mINFO [2024-07-08 14:30:08 - session.py:294] - NetworkTask started[0m
[1;92mINFO [2024-07-08 14:30:08 - session.py:139] - Session initialized: Layer 158[0m
[1;92mINFO [2024-07-08 14:30:08 - session.py:140] - Device: CPython 3.10.12 - Pyrogram 2.0.106[0m
[1;92mINFO [2024-07-08 14:30:08 - session.py:141] - System: Linux 5.15.0-101-generic (en)[0m
[1;92mINFO [2024-07-08 14:30:08 - session.py:155] - Session started[0m
[1;92mINFO [2024-07-08 14:30:08 - session.py:272] - PingTask started[0m
[1;92mINFO [2024-07-08 14:30:09 - client.py:698] - [vid-compressbot] [LOAD] MessageHandler("user_not_join") in group 0 from "plugins.user_entry"[0m
[1;92mINFO [2024-07-08 14:30:09 - client.py:775] - [vid-compressbot] Successfully loaded 1 plugin from "plugins"[0m
[1;92mINFO [2024-07-08 14:30:09 - dispatcher.py:154] - Started 8 HandlerTasks[0m
[1;91mERROR [2024-07-08 14:30:11 - dispatcher.py:253] - 'list' object has no attribute 'write'[0m
Traceback (most recent call last):
  File "/home/freeguy/.local/lib/python3.10/site-packages/pyrogram/dispatcher.py", line 240, in handler_worker
    await handler.callback(self.client, *args)
  File "/home/freeguy/Desktop/project/vid-compress/app/plugins/user_entry.py", line 21, in user_not_join
    await bot.send_message(msg.from_user.id, text=join_text, reply_markup = btn.join_channel(lang=user.lang , url=setting.channel_url))
  File "/home/freeguy/.local/lib/python3.10/site-packages/pyrogram/methods/messages/send_message.py", line 134, in send_message
    reply_markup=await reply_markup.write(self) if reply_markup else None,
AttributeError: 'list' object has no attribute 'write'
[1;92mINFO [2024-07-08 14:30:33 - idle.py:75] - Stop signal received (SIGINT). Exiting...[0m
[1;92mINFO [2024-07-08 14:30:36 - tcp.py:67] - Using proxy 127.0.0.1[0m
[1;92mINFO [2024-07-08 14:30:36 - connection.py:47] - Connecting...[0m
[1;92mINFO [2024-07-08 14:30:37 - connection.py:54] - Connected! Production DC4 - IPv4[0m
[1;92mINFO [2024-07-08 14:30:37 - session.py:294] - NetworkTask started[0m
[1;92mINFO [2024-07-08 14:30:37 - session.py:139] - Session initialized: Layer 158[0m
[1;92mINFO [2024-07-08 14:30:37 - session.py:140] - Device: CPython 3.10.12 - Pyrogram 2.0.106[0m
[1;92mINFO [2024-07-08 14:30:37 - session.py:141] - System: Linux 5.15.0-101-generic (en)[0m
[1;92mINFO [2024-07-08 14:30:37 - session.py:155] - Session started[0m
[1;92mINFO [2024-07-08 14:30:37 - session.py:272] - PingTask started[0m
[1;92mINFO [2024-07-08 14:30:38 - client.py:698] - [vid-compressbot] [LOAD] MessageHandler("user_not_join") in group 0 from "plugins.user_entry"[0m
[1;92mINFO [2024-07-08 14:30:38 - client.py:775] - [vid-compressbot] Successfully loaded 1 plugin from "plugins"[0m
[1;92mINFO [2024-07-08 14:30:38 - dispatcher.py:154] - Started 8 HandlerTasks[0m
[1;91mERROR [2024-07-08 14:30:40 - dispatcher.py:253] - 'list' object has no attribute 'write'[0m
Traceback (most recent call last):
  File "/home/freeguy/.local/lib/python3.10/site-packages/pyrogram/dispatcher.py", line 240, in handler_worker
    await handler.callback(self.client, *args)
  File "/home/freeguy/Desktop/project/vid-compress/app/plugins/user_entry.py", line 22, in user_not_join
    await bot.send_message(msg.from_user.id, text=join_text, reply_markup = btn.join_channel(lang=user.lang , url=setting.channel_url))
  File "/home/freeguy/.local/lib/python3.10/site-packages/pyrogram/methods/messages/send_message.py", line 134, in send_message
    reply_markup=await reply_markup.write(self) if reply_markup else None,
AttributeError: 'list' object has no attribute 'write'
[1;92mINFO [2024-07-08 14:31:28 - idle.py:75] - Stop signal received (SIGINT). Exiting...[0m
[1;92mINFO [2024-07-08 14:31:29 - tcp.py:67] - Using proxy 127.0.0.1[0m
[1;92mINFO [2024-07-08 14:31:29 - connection.py:47] - Connecting...[0m
[1;92mINFO [2024-07-08 14:31:30 - connection.py:54] - Connected! Production DC4 - IPv4[0m
[1;92mINFO [2024-07-08 14:31:30 - session.py:294] - NetworkTask started[0m
[1;92mINFO [2024-07-08 14:31:30 - session.py:139] - Session initialized: Layer 158[0m
[1;92mINFO [2024-07-08 14:31:30 - session.py:140] - Device: CPython 3.10.12 - Pyrogram 2.0.106[0m
[1;92mINFO [2024-07-08 14:31:30 - session.py:141] - System: Linux 5.15.0-101-generic (en)[0m
[1;92mINFO [2024-07-08 14:31:30 - session.py:155] - Session started[0m
[1;92mINFO [2024-07-08 14:31:30 - session.py:272] - PingTask started[0m
[1;92mINFO [2024-07-08 14:31:31 - client.py:698] - [vid-compressbot] [LOAD] MessageHandler("user_not_join") in group 0 from "plugins.user_entry"[0m
[1;92mINFO [2024-07-08 14:31:31 - client.py:775] - [vid-compressbot] Successfully loaded 1 plugin from "plugins"[0m
[1;92mINFO [2024-07-08 14:31:31 - dispatcher.py:154] - Started 8 HandlerTasks[0m
[1;92mINFO [2024-07-08 14:32:38 - idle.py:75] - Stop signal received (SIGINT). Exiting...[0m
[1;92mINFO [2024-07-08 14:33:13 - tcp.py:67] - Using proxy 127.0.0.1[0m
[1;92mINFO [2024-07-08 14:33:13 - connection.py:47] - Connecting...[0m
[1;92mINFO [2024-07-08 14:33:13 - connection.py:54] - Connected! Production DC4 - IPv4[0m
[1;92mINFO [2024-07-08 14:33:13 - session.py:294] - NetworkTask started[0m
[1;92mINFO [2024-07-08 14:33:14 - session.py:139] - Session initialized: Layer 158[0m
[1;92mINFO [2024-07-08 14:33:14 - session.py:140] - Device: CPython 3.10.12 - Pyrogram 2.0.106[0m
[1;92mINFO [2024-07-08 14:33:14 - session.py:141] - System: Linux 5.15.0-101-generic (en)[0m
[1;92mINFO [2024-07-08 14:33:14 - session.py:155] - Session started[0m
[1;92mINFO [2024-07-08 14:33:14 - session.py:272] - PingTask started[0m
[1;92mINFO [2024-07-08 14:33:14 - client.py:698] - [vid-compressbot] [LOAD] MessageHandler("user_not_join") in group 0 from "plugins.user_entry"[0m
[1;92mINFO [2024-07-08 14:33:14 - client.py:775] - [vid-compressbot] Successfully loaded 1 plugin from "plugins"[0m
[1;92mINFO [2024-07-08 14:33:14 - dispatcher.py:154] - Started 8 HandlerTasks[0m
[1;92mINFO [2024-07-08 14:33:49 - idle.py:75] - Stop signal received (SIGINT). Exiting...[0m
